input/docs/getting-started/upgrade.md

```diff
@@ -44,6 +44,151 @@ To update Cake follow these instructions
     </div>
 </div>
 
+# Cake 1.x to Cake 2.0
+
+Cake 2.0 is a major version containing breaking changes.
+
+## Supported runners and required .NET version
+
+To allow Cake to make use of modern platform features, make life easier for extension authors and simplify decision process of users
+we have decided to stop shipping [Cake runner for .NET Framework] and the already deprecated [Cake runner for .NET Core] with 2.0.
+Additionally we dropped support to run on .NET Core 2.1 and .NET Core 3.0 for [Cake .NET Tool] and [Cake Frosting].
+
+This means that, starting with Cake 2.0, you will need to have the .NET SDK installed on your build machine, at a minimum .NET Core 3.1,
+but .NET 6 is recommended, in order to **run** Cake.
+In other words, Cake itself will no longer **run** on .NET Framework, Mono and .NET Core 3.0 or older.
+Cake will continue to support building of .NET Framework projects as well as projects targeting .NET Core 3.0 or older.
+
+Supported platform matrix for Cake 2.0 will look like this:
+
+| Runner                           | .NET 6 | .NET 5 | .NET Core 3.1 |
+| -------------------------------- |:------:|:------:|:-------------:|
+| [Cake .NET Tool]                 | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> |
+| [Cake Frosting]                  | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> |
+
+As a consequence of no longer shipping [Cake runner for .NET Framework] we will also stop shipping the [Cake.Portable Chocolatey package]
```

Comment:
@pascalberger @augustoproiete Am I right in saying that these (the Chocolatey and Brew) packages won't be pushed out as part of the RC release?

Reply:
Yes, we no longer will push to Chocolatey and Brew